aesara/link/jax/dispatch.py

```diff
@@ -831,16 +823,6 @@ def qr_full(x, mode=mode):
     return qr_full
 
 
-@jax_funcify.register(QRIncomplete)
```

Comment:
Why do you remove this?

Reply:
That `Op` was redundant, so it was removed.